### ISSUE TITLE
[FEAT][Multiple Tools in Parallel for the `Flow` class

### ISSUE THREAD

**Issue by ZackBradshaw** [kyegomez/swarms#114]
## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/kyegomez) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/kyegomez/swarms/issues/114">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/kyegomez/swarms/issues/114/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/kyegomez/swarms/issues/114/pledge.svg">
</picture>
</a>

**Comment by ZackBradshaw** [comment]
This is on hold till #85  is merged this has the capability to read these formats through more testing and documentation will be required

**Comment by evelynmitchell** [comment]
Can this be closed?
